packages/dd-trace/test/appsec/iast/taint-tracking/sources/taint-tracking.express.plugin.spec.js

```diff
@@ -208,12 +209,8 @@ describe('Path params sourcing with express', () => {
         res.status(200).send()
       })
 
-      app.param((param, option) => {
-        return checkParamIsTaintedAndNext
-      })
-
-      app.param('parameter1')
-      app.param('parameter2')
+      app.param('parameter1', checkParamIsTaintedAndNext)
+      app.param('parameter2', checkParamIsTaintedAndNext)
```

Comment:
Could it be written like this?
```suggestion
      app.param(['parameter1', 'parameter2'], checkParamIsTaintedAndNext)
```
https://expressjs.com/en/5x/api.html#app.param

Reply:
Done